### ISSUE TITLE
Crashes on macOS 10.12 (Sierra)

### ISSUE THREAD

**Issue by tvandervossen** [Fingertips/ColorPicker#3]

**Comment by gordio** [comment]
Details: http://pastie.org/private/td2fkodj3ub7iviqycx2zw

**Comment by tvandervossen** [comment]
Yeah, we know. Apple made some major API changes to the color panel. It’s unlikely that we’ll find the time to look into this anytime soon.
